modules/pv.py

```diff
@@ -35,26 +39,44 @@
         description: Disks from which the Physical Volumes are to be created,
                      or Physical Volumes that are to be removed needs to be
                      specified here.
-    options:
-        required: false
-        description: Extra options that needs to be passed while creating the
-                     Physical Volumes can be given here. Check the man page of
-                     pvcreate for more info.
     size:
-        required: true if action is resize and opertion is shrink
+        required: true
         description: Specifies to what size the physical volume is to be
                      shrunken
+    force:
+        required: true
+        description: Force  the  creation  without  any confirmation.
+    uuid:
+        required: true
+        description: Specify the uuid for the device.  Without this option,
+                     pvcreate generates a random uuid.
+    zero:
+        required: true
+        description: Whether or not the first 4 sectors of the device should
+                     be wiped.
+    metadatasize:
+        required: true
+        description: The approximate amount of space to be set aside for
+                     each metadata area.
+    dataalignment:
+        required: true
+        description: Align the start of the data to a multiple of this number.
 
-authors: Anusha Rao, Nandaja Varma
 '''
 
 EXAMPLES = '''
 #Create Physical Volumes /dev/sdb and /dev/sdc with
 #dataalignment 1280k
     - pv: action=create disks='["/dev/sdb", "/dev/sdc"]'
           options="--dataalignment 1280k"
+          force=['y']
```

Comment:
The value is not provided like ['y'], we give such values only when we have lists. Just y or yes will do. For example:

force=yes